src/Libraries/RevitNodes/Elements/Views/View3D.cs

```diff
@@ -1,16 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
-
 using Autodesk.DesignScript.Runtime;
 using Autodesk.Revit.DB;
 using RevitServices.Persistence;
 using RevitServices.Transactions;
 
 namespace Revit.Elements.Views
 {
-    //[SupressImportIntoVM]
```

Comment:
it was commented out and I just deleted that comment. as for the using statement, it was not used anywhere in the code, why import it?